src/client.ts

```diff
@@ -121,27 +119,27 @@ export default class Client {
       this.waitingSubscriptions[subId] = true;
       setTimeout( () => {
         if (this.waitingSubscriptions[subId]){
-          handler(new Error('Subscription timed out - no response from server'));
+          handler([new Error('Subscription timed out - no response from server')]);
```

Comment:
Note that a bug found and fixed in here.
handler used once with Error and then with Error[].
changed to Error[].